### PR TITLE
Ticket 4

### PR DESCRIPTION
**Pull request by MSalim16**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+describe("/api/reviews", () => {
+  test("GET - status: 200, responds with an array of review objects", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body: { reviews } }) => {
+        expect(reviews.length).toEqual(13);
+        reviews.forEach((review) => {
+          expect(review).toHaveProperty("comment_count");
```
Comment (corkesw) [inline __tests__/app.test.js]: I'm not sure why this assertion is not just part of the matched object where we could also assert the data-type of it's value

File: __tests__/app.test.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+          expect(review).toMatchObject({
+            title: expect.any(String),
+            designer: expect.any(String),
+            owner: expect.any(String),
+            review_img_url: expect.any(String),
+            review_body: expect.any(String),
+            category: expect.any(String),
+            votes: expect.any(Number),
```
Comment (corkesw) [inline __tests__/app.test.js:51]: The README states that the objects should also have  created_at and review_id properties which we have not tested for.

File: __tests__/app.test.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+          });
+        });
+      });
+  });
+});
```
Comment (corkesw) [inline __tests__/app.test.js:59]: We need to also check that the reviews are coming back in the default sort order ie, by date, descending

File: __tests__/app.test.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+const testData = require("../db/data/test-data/index.js");
+const seed = require("../db/seeds/seed.js");
+require("jest-sorted");
```
Comment (corkesw) [inline __tests__/app.test.js]: As per the last ticket, this line is not doing anything

File: app.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const { getCategories } = require("./controllers/categories");
+const { getReviews } = require("./controllers/reviews");
+const app = express();
+
+app.use(express.json());
```
Comment (corkesw) [inline app.js]: This middleware should not be included. You should remove line 6

**Review by corkesw — COMMENTED** [review]
There is a little bit more to be done with the testing here. Please action the comments before proceeding.

File: app.js (hunk 5 above)
Comment (MSalim16) [reply app.js]: I removed it sorry its because i did another pr too quick before the first got approved

**Pull request merged**